src/app/profiles/App/ContextPopin/withConnect.ts

```diff
@@ -0,0 +1,17 @@
+import { ProfilesState } from '../../store/reducers';
+import { getContextPopinContent } from '../../store/selectors/contextPopin';
+import { setContextPopin } from '../../store/actions/contextPopin';
+import { ContextPopinState } from '../../store/reducers/contextPopin.reducer';
+import { Dispatch } from 'redux';
+import { connect } from 'react-redux';
+
+const mapStateToProps = (state: ProfilesState) => ({
+  contextPopin: getContextPopinContent(state)
```

Comment:
I guess what you want is `getContextPopinState` ?

Reply:
hum, I need the contents here so getContextPopinContent